### DISCUSSION TITLE
`module.exports` "format"?

### DISCUSSION THREAD

**Discussion by jakubmazanec** [parcel-bundler/parcel#8233] [Q&A] [answered]
Does Parcel have setting like Rollup's https://rollupjs.org/guide/en/#outputexports ? It seems to me that if I try bundling simple library:
```json5
// package.json
{
  "name": "my-library",
  "version": "1.0.0",
  "source": "src/index.js",
  "main": "dist/main.js",
  "devDependencies": {
    "parcel": "latest"
  }
}
```
```js
// src/index.js
export default function add(a, b) {
  return a + b;
}
```

Parcel always outputs code that adds properties to `module.exports`, e.g. 
```js
// dist/main.js
/* ... */
$parcel$export(module.exports, "default", () => $4fa36e821943b400$export$2e2bcd8739ae039);
function $4fa36e821943b400$export$2e2bcd8739ae039(a, b) {
    return a + b;
}
/* ... */
```
But I want `$4fa36e821943b400$export$2e2bcd8739ae039` to be directly assigned to `module.exports`, because sometimes it is needed (e.g. ESLint expects that package with a config export it this way).

**Comment by mischnic** [comment, marked as answer]
You should be able to do `module.exports = function add() { ... }` in your source and get the result you want

**Reply by jakubmazanec** [reply]
So the answer is no 😢 But thanks for the reply.
